lnrpc/rest-annotations.yaml

```diff
@@ -261,6 +261,9 @@ http:
     - selector: walletrpc.WalletKit.NextAddr
       post: "/v2/wallet/address/next"
       body: "*"
+    - selector: walletrpc.WalletKit.ListWalletAddresses
+      post: "/v2/wallet/addresses/list"
```

Comment:
REST URIs should be singular in my opinion, so this should just be `/v2/wallet/address/list`.
Also, because there are no input parameters, I think this can be a `get` method (the above `NextAddr` should be `get` as well, but we can't change that now as it would break existing users, but I'd like to at least have it correct for new RPCs).